Don't check for EGL now

This patch can be removed when we update to the latest version 
and have all dependencies, like cairo, building with EGL.

--- a/Source/cmake/OptionsGTK.cmake	2017-04-29 15:18:54.753838020 -0700
+++ b/Source/cmake/OptionsGTK.cmake	2017-04-29 16:17:08.276972910 -0700
@@ -47,7 +47,6 @@
 find_package(WebP REQUIRED)
 find_package(ATSPI 2.5.3)
 find_package(GObjectIntrospection)
-find_package(EGL)
 find_package(GeoClue2 2.1.5)
 find_package(GnuTLS 3.0.0)
 find_package(LibNotify)
